_ru/scala3/book/taste-repl.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: multipage-overview
+title: REPL
+scala3: true
+partof: scala3-book
+overview-name: "Scala 3 — Book"
+type: chapter
```

Comment:
```suggestion
type: section
```
be careful to replicate the `type` for each page (there are others) - this is what generates the hierarchical sidebar